test-app/package.json

```diff
@@ -74,6 +74,7 @@
     "broccoli-asset-rev": "^3.0.0",
     "concurrently": "^7.6.0",
     "ember-a11y-testing": "^5.2.0",
+    "ember-async-data": "^0.7.0",
```

Comment:
Adding this should not be necessary, but it fixes a weird issue making Embroider with _safe_ settings not fail the build, see https://github.com/embroider-build/embroider/issues/1370